scripts/patch-harvester

```diff
@@ -14,4 +14,26 @@ yq e -i '.appVersion = "'"${APP_VERSION}"'"' ./deploy/charts/harvester-crd/Chart
 echo "Replace the Harvester image tag to ${IMAGE_PUSH_TAG}"
 yq e -i '.containers.apiserver.image.tag = "'"${IMAGE_PUSH_TAG}"'"' ./deploy/charts/harvester/values.yaml
 yq e -i '.webhook.image.tag = "'"${IMAGE_PUSH_TAG}"'"' ./deploy/charts/harvester/values.yaml
-yq e -i '.upgrade.image.tag = "'"${IMAGE_PUSH_TAG}"'"' ./deploy/charts/harvester/values.yaml
\ No newline at end of file
+yq e -i '.upgrade.image.tag = "'"${IMAGE_PUSH_TAG}"'"' ./deploy/charts/harvester/values.yaml
+
+echo "Replace the Harvester image repo to ${REPO}"
+replace_image_repo()
```

Comment:
nit: Can we move function definitions to the top?

Reply:
updated